rosmon_core/src/launch/launch_config.h

```diff
@@ -5,6 +5,7 @@
 #define ROSMON_LAUNCH_LAUNCH_CONFIG_H
 
 #include "node.h"
+#include "../fmt_no_throw.h"
```

Comment:
Do not use relative links, its possible they will be broken in catkin install mode. Always try to use the path from package level.

Reply:
Hmm seems like rosmon core follows this pattern, bad design but since its an external fork lets stick with their convention